modules/utils.lua

```diff
@@ -910,7 +914,7 @@ else
         -- Clear Extras
         for i = 1, 20 do
             if DoesExtraExist(vehicle, i) then
-                SetVehicleExtra(vehicle, i, true)
+                SetVehicleExtra(vehicle, i, false)
```

Comment:
consider extracting a local function clearVehicleExtras to reduce duplicate code

Reply:
How about something like this?

`    ---Clear all vehicle extras
    ---@param vehicle integer
    function ClearAllVehicleExtras(vehicle)
        for i = 1, 20 do
            if DoesExtraExist(vehicle, i) then
                SetVehicleExtra(vehicle, i, false)
            end
        end
    end

    ---Set the status of an extra on the vehicle
    ---@param vehicle integer
    ---@param extra integer
    ---@param enable boolean
    function ChangeVehicleExtra(vehicle, extra, enable)
        if not DoesExtraExist(vehicle, extra) then return end
        local isExtraOn = IsVehicleExtraTurnedOn(vehicle, extra)

        if enable ~= isExtraOn then
            SetVehicleExtra(vehicle, extra, not enable)
        end
    end

    ---Set the vehicle extras of a vehicle according to a table
    ---@param vehicle integer
    ---@param extras table<integer, boolean>
    function SetVehicleExtras(vehicle, extras)
        ClearAllVehicleExtras(vehicle)
        
        for id, enabled in pairs(extras) do
            ChangeVehicleExtra(vehicle, tonumber(id) --[[@as integer]], enabled)
        end
    end`